examples/config.yml

```diff
@@ -9,8 +9,9 @@ server:
   actionCacheReadOnly: false
   port: 8980
   grpcMetrics:
-    enabled: false
-    provideLatencyHistograms: false
+    enabled: true
```

Comment:
any reason to have lines 12:13 and 95:96 be different?

Reply:
Nothing specific, I was testing locally and updated to ensure both combination works.